applications/network_radar_pipeline/cpp/CMakeLists.txt

```diff
@@ -0,0 +1,66 @@
+# SPDX-FileCopyrightText: Copyright (c) 2023 NVIDIA CORPORATION & AFFILIATES. All rights reserved.
+# SPDX-License-Identifier: Apache-2.0
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+cmake_minimum_required(VERSION 3.20)
+project(network_radar_pipeline CXX)
+
+find_package(holoscan 0.6 REQUIRED CONFIG
+             PATHS "/opt/nvidia/holoscan" "/workspace/holoscan-sdk/install")
+
+# Enable cuda language
+set(CMAKE_CUDA_ARCHITECTURES "70;80")
+enable_language(CUDA)
+
+# Download MatX
+include(FetchContent)
+FetchContent_Declare(
+  MatX
+  GIT_REPOSITORY https://github.com/NVIDIA/MatX.git
+  GIT_TAG v0.3.0
```

Comment:
This is a pretty old version, but I think you said you updated already.

Reply:
@dylan-eustice can you address Cliff's comment?

Reply:
@wendell-hom There's an updated branch that has a newer MatX version that we can update the PR with

Reply:
thanks @dylan-eustice, could you give it a test and see if everything is running smoothly.

Reply:
He's run this and everything seems to be working